modules/core/shared/src/main/scala/eu/timepit/refined/numeric.scala

```diff
@@ -181,3 +182,84 @@ private[refined] trait NumericInference {
   ): Greater[A] ==> Greater[B] =
     Inference(nc.gt(wa.value, nc.fromInt(tb())), s"greaterInferenceWitNat(${wa.value}, ${tb()})")
 }
+
+private[refined] trait NumericMin {
+  implicit val byteMin: Min[Byte] = Min.instance(Byte.MinValue)
+  implicit val shortMin: Min[Short] = Min.instance(Short.MinValue)
+  implicit val intMin: Min[Int] = Min.instance(Int.MinValue)
+  implicit val longMin: Min[Long] = Min.instance(Long.MinValue)
+  implicit val floatMin: Min[Float] = Min.instance(Float.MinValue)
+  implicit val doubleMin: Min[Double] = Min.instance(Double.MinValue)
+  implicit val charMin: Min[Char] = Min.instance(Char.MinValue)
```

Comment:
I would definitely move these into the `Min` companion since the are not related to the predicates in `numeric`.